containers/Ideas/Idea.js

```diff
@@ -313,6 +331,12 @@ export const Idea = (props) => {
     }
   }, [isEditing])
 
+  useEffect(() => {
```

Comment:
What's the advantage/reason of putting the error handling in a hook instead of doing it where the error happens?

Reply:
the reason is, that we use the hook way in IdeaCreate:
https://github.com/liqd/zt-app/blob/bfbac203a16e65da3c7b55f41267fdf6ecd8683e/containers/Ideas/IdeaCreate.js#L51

and the advantage is that it "centralizes" the `Alert.alert()` call. If there will be more error throws in future we can just set the `error` state variable. Imho makes it more consistent and easier to extend. Does that make sense to you?

Reply:
ok, didn't know we do it that way elsewhere too. I was just a bit concerned with state being async and everything that it makes it a bit more complicated, it could also be centralized in a function as long as we don't pass it down to subcomponents. I'm also fine with the hook though!

Reply:
yes that is true. I just saw that we also use the hook way for the `LoginScreen`. If you don't mind, I would leave it for now like that and revisit the error handling, when we work on this issue?

https://github.com/liqd/zt-app/issues/378

Reply:
sounds good to me, thanks :)